apache2/waf_lock/waf_lock.cpp

```diff
@@ -72,7 +67,31 @@ int lock_create(struct waf_lock *new_lock, struct waf_lock_args *new_lock_args)
         }
         return WAF_SUCCESS_LOCK_OPEN;
     }
+
+    // Should never get here
+    return WAF_SUCCESS_LOCK_CREATE;
+}
 #else
+int lock_create(struct waf_lock *new_lock, struct waf_lock_args *new_lock_args) {
```

Comment:
Just add a comment here // Windows to be explicit about these blocks. Coordinating `#ifdef` is pretty painful.